commons/all/src/main/java/org/infinispan/commons/util/TypedProperties.java

```diff
@@ -88,14 +88,10 @@ public long getLongProperty(String key, long defaultValue, boolean doStringRepla
       }
    }
 
-   public long getDurationProperty(String key, long defaultValue) {
```

Comment:
Public API for users, do not remove.

Reply:
we need to add a test then, since the code tells me it's not used

Reply:
It's not being tested in `TypedPropertiesTest`. Please do.